src/test/java/io/confluent/examples/streams/FanoutLambdaIntegrationTest.java

```diff
@@ -72,6 +72,7 @@ public void shouldFanoutTheInput() {
     streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "fanout-lambda-integration-test");
     streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy config");
     streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
+    streamsConfiguration.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
```

Comment:
@lct45  Ok sorry for the back and forth, not sure if you saw my last comment but let's actually just leave it as a ByteArraySerde for now and I can merge it

Reply:
Ah sorry @ableegoldman just responded - it working with byte array and string seems slightly concerning to me (lmk what you think tho). I can't dive into this today but can tomorrow

Reply:
I'm not surprised it would work with both, an IntegerSerde for example can tell when the data type is not right because it has to always be 4 bytes, whereas a StringSerde could happily interpret any variable length series of bytes as a string (it's just that the string may be complete nonsense).

So let's just set it to be a ByteArraySerde and leave it at that. Sorry for making this confusing 😅

Reply:
If you get the chance to change it back into a ByteArraySerde today then I can merge it, otherwise if you get to it tomorrow then can you just merge it since I'll be on a plane?

Reply:
No worries, thanks for diving into this. just updated